### PR TITLE
Update versions in main

### PR DESCRIPTION
**Pull request by rturner3**

### CODE REVIEW

**Review by azdagron — APPROVED** [review]
Thanks, @rturner3 !

File: test/integration/suites/upgrade/versions.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@
 1.4.2
 1.4.3
 1.4.4
+1.4.5
```
Comment (amartinezfayo) [inline test/integration/suites/upgrade/versions.txt:6]: 1.5.0 is missing here.

Comment (rturner3) [reply test/integration/suites/upgrade/versions.txt:6]: Ah, you're right. I always get confused which versions belong in here.

**Review by MarcosDY — APPROVED** [review]
LGTM!

**Pull request merged**